### PR TITLE
AHWR-315: Added support for usage of new env var for multi species

### PR DESCRIPTION
**Pull request by RobCatton**
## Description of changes

Added support for usage of new env var for multi species, in helm chart file, docker compose, and app config js file.

### CODE REVIEW

File: app/config/index.js [hunk 1]
```diff
@@ -94,6 +97,9 @@ const config = {
   },
   endemics: {
     enabled: process.env.ENDEMICS_ENABLED
+  },
+  multiSpecies: {
+    enabled: process.env.MULTI_SPECIES_ENABLED
```
Comment (mattyod) [inline app/config/index.js:102]: should this, (and endemics above now it's required) be asserting that the value is 'true'? e.g.
```suggestion
    enabled: process.env.MULTI_SPECIES_ENABLED === 'true'
```

Comment (RobCatton) [reply app/config/index.js:102]: good catch! thats my bad - this is why we have code reviews!

Comment (RobCatton) [reply app/config/index.js:102]: I guess the endemics enabled also is wrong, and presumably has been for a while?

Comment (RobCatton) [reply app/config/index.js:102]: This has opened a can of worms - there are loads of instances of this, across a few repos. It seems to be 'covered' by the schema having defaults, but it means that the env vars arent being used properly.

**Review by aaroncarroll82 — APPROVED** [review]

**Pull request merged**